### PR TITLE
Follow up from 1056

### PR DESCRIPTION
**Pull request by wojtek-t**
Addresses @mm4tt  post-submit comments in #1056 (which was much easier given recent refactorings).

### CODE REVIEW

File: clusterloader2/cmd/clusterloader.go [hunk 1]
```diff
@@ -71,6 +71,7 @@ func initClusterFlags() {
 	flags.StringSliceEnvVar(&clusterLoaderConfig.ClusterConfig.MasterInternalIPs, "master-internal-ip", "MASTER_INTERNAL_IP", nil /*defaultValue*/, "Cluster internal/private IP of the master vm, supports multiple values when separated by commas")
 	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.KubemarkRootKubeConfigPath, "kubemark-root-kubeconfig", "KUBEMARK_ROOT_KUBECONFIG", "",
 		"Path the to kubemark root kubeconfig file, i.e. kubeconfig of the cluster where kubemark cluster is run. Ignored if provider != kubemark")
+	flags.BoolEnvVar(&clusterLoaderConfig.ClusterConfig.APIServerPprofByClientEnabled, "apiserver-pprof-by-client-enabled", "APISERVER_PPROF_BY_CLIENT_ENABLED", true, "Whether apserver pprof endpoint can be accessed by Kubernetes client.")
```
Comment (mm4tt) [inline clusterloader2/cmd/clusterloader.go:74]: s/apserver/apiserver/

Comment (wojtek-t) [reply clusterloader2/cmd/clusterloader.go]: Done

**Comment by wojtek-t** [comment]
@mm4tt - PTAL

**Comment by mm4tt** [comment]
/lgtm

Thanks!

**Pull request merged**